app/models/giveaway.rb

```diff
@@ -13,12 +12,16 @@ def choose_winners
                 return 
             else
                 w.item_won = product[:prod_id]
-                winners << w
+                w.save
             end
         end
       end
     end
 
+    def winners 
+        users.where("item_won <> ''")
```

Comment:
Nitpick: If you set this up as a scope on the `User` model, this would just be
```
users.winners
```
Depends whether you think you would ever use the scope outside this scenario.